src/map/atcommand.cpp

```diff
@@ -2804,12 +2804,12 @@ ACMD_FUNC(zeny)
 	}
 
 	if(zeny > 0){
-	    if((ret=pc_getzeny(sd,zeny,LOG_TYPE_COMMAND,NULL)) == 1)
+	    if((ret=pc_getzeny(sd,zeny,LOG_TYPE_COMMAND)) == 1)
```

Comment:
Not an issue of your PR, but for charcommands this will wrong data.
We should create an issue for this and fix it in another PR, as it requires to revise the function signature of all atcommands.

Reply:
How will it create wrong data? The player is gaining zeny from a command, it would be trivial to search uses of the `#zeny` char command

Reply:
It did create wrong data right now, because right now it would look like you got zeny via atcommand even though you did not have the `@zeny` command.

But yeah it is just as trivial as to join 2 mail related log entries and still we redesign it.